src/crimson/os/seastore/extentmap_manager/btree/extentmap_btree_node_impl.cc

```diff
@@ -73,7 +73,7 @@ ExtMapInnerNode::rm_lextent(ext_context_t ec, objaddr_t lo, lext_map_val_t val)
   auto rm_pt = get_containing_child(lo);
   return extmap_load_extent(ec, rm_pt->get_val(),  get_meta().depth - 1).safe_then(
     [this, ec, rm_pt, lo, val=std::move(val)](auto extent) mutable {
-    if (extent->at_min_capacity()) {
+    if (extent->at_min_capacity() && get_node_size() != 1) {
```

Comment:
could you move this change into a separate commit? also, does it make sense to change like
```c++
if (extent->at_min_capacity() && get_node_size() > 1) {
```

to help reader to understand that we need another entry in the internal node to from which the pointers/elements are transferred.